diff --git a/node_modules/@types/react-simple-maps/index.d.ts b/node_modules/@types/react-simple-maps/index.d.ts
index 00e4837..b24275e 100644
--- a/node_modules/@types/react-simple-maps/index.d.ts
+++ b/node_modules/@types/react-simple-maps/index.d.ts
@@ -123,6 +123,8 @@ interface ZoomPanProps {
 
 interface GeographiesChildrenArgument {
     geographies: any[];
+    outline: any;
+    borders: any;
     path: GeoPath;
     projection: GeoProjection;
 }
